app/javascript/packages/document-capture/components/unknown-error.tsx

```diff
@@ -33,9 +43,16 @@ function UnknownError({
       </p>
     );
   }
-  if (err) {
+  if (err && !hasDismissed) {
     return <p key={err.message}>{err.message}</p>;
   }
+  if (err && hasDismissed) {
+    return (
+      <p key={err.message}>
+        {err.message} <a href={helpCenterLink}>{t('doc_auth.info.review_examples_of_photos')}</a>
```

Comment:
Elsewhere in the application, we almost universally make help center links open in a new window and render as an external link (usually using the [`<Link />` component](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/components/link.tsx) in React). Should we do that here?

Reply:
Makes sense!  I okayed this with @kellular and made the change.